src/tests/ftest/cart/threaded_server.c

```diff
@@ -140,11 +150,13 @@ int main(int argc, char **argv)
 	D_MUTEX_UNLOCK(&lock);
 	printf("Stop rpc exited with rc = %d\n", rc);
 
+	rc = 0;
+stop:
 	status = STOP;
 	printf("Waiting for threads to stop\n");
 
-	for (rc = 0; rc < NUM_THREADS; rc++)
-		pthread_join(thread[rc], NULL);
+	for (i = 0; i < thread_count; i++)
+		pthread_join(thread[i], NULL);
```

Comment:
Do we need to check the return value of `pthread_join`?

Reply:
coverity doesn't complain about it so I'm going to punt it til later.  There isn't much we would do with that error.